src/app/screens/App/screens/Home/strings.js

```diff
@@ -3,3 +3,4 @@ export const AUTHOR = 'Autor';
 export const SELECTFILTER = 'Seleccionar filtro';
 export const SEARCH = 'Buscar...';
 export const SEARCHBUTTONALTTEXT = 'Buscar';
+export const BOOKCOVERALTTEXT = 'Libro';
```

Comment:
my mistake. 
user UPPER_SNAKE_CASE for constants.
UPPERCASE + snake_case    like  BOOK_COVER_ALT_TEXT